scripts/generate-conda-lockfile.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+REQS_DIR="$(pwd)/conda-reqs"
```

Comment:
This should be more robust so that it can be called in multiple places. I would use the following: https://stackoverflow.com/questions/59895/how-do-i-get-the-directory-where-a-bash-script-is-located-from-within-the-script